### ISSUE TITLE
[plugin|tiling] Centre again on window destroyed same space

### ISSUE THREAD

**Issue by choco** [koekeishiya/chunkwm#79]
This isn't a bug, but wanted to report it since it didn't look super good ahah. Right now when an app has multiple windows in the same space, when a window is destroyed macOS will automatically focus the other open window. 
But since the window destroyed notification for window1 is sent after the other window, window2, is focused, we have that:
1. cursor in centred in window2
2. window2 is actually destroyed, removed from window tree
3. the centre of the window2 since it may get resized during retiling
Basically we end up with the cursor non centred on the focused window after another one of the same app is destroyed on the same space.
